internal/query/app.go

```diff
@@ -368,6 +369,95 @@ func (q *Queries) ProjectByClientID(ctx context.Context, appID string) (project
 	return project, err
 }
 
+//go:embed app_oidc_project_permission.sql
+var appOIDCProjectPermissionQuery string
+
+func (q *Queries) CheckProjectPermissionByClientID(ctx context.Context, clientID, userID string) (_ bool, err error) {
+	ctx, span := tracing.NewSpan(ctx)
+	defer func() { span.EndWithError(err) }()
+
+	var p *projectPermission
+	err = q.client.QueryRowContext(ctx, func(row *sql.Row) error {
+		p, err = scanProjectPermissionByClientID(row)
+		return err
+	}, appOIDCProjectPermissionQuery,
+		authz.GetInstance(ctx).InstanceID(),
+		clientID,
+		domain.AppStateActive,
+		domain.ProjectStateActive,
+		userID,
+		domain.UserStateActive,
+		domain.ProjectGrantStateActive,
+		domain.UserGrantStateActive,
+	)
+	return p.permission(), err
+}
```

Comment:
based on the comments in other places, we could just scan for the booleans and return them both

Reply:
Removed all values besides the booleans, which get returned